docs/Tasks.txt

```diff
@@ -71,7 +71,8 @@ SYSTEMDS-100 Various Fixes
 SYSTEMDS-110 New Builtin Functions
  * 111 Time builtin function for script-level measurements            OK
  * 112 Image data augmentation builtin functions                      OK
- 
+ * 113 Builtin functions for linear regression algorithms             OK
```

Comment:
Please, try to avoid leaking files across PRs.

Reply:
I will watch out for that in the future.